### PR TITLE
Include Bazaar digest in legacy charm-info API endpoint.

### PR DESCRIPTION
**Pull request by frankban**
Also activate the legacy server in charmd.

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results: http://ci.jujugui.org:8080/job/charmstore/268/

**Comment by mhilton** [comment]
:+1: LGTM

File: cmd/charmd/main.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ func serve() error {
 		AuthUsername: conf.AuthUsername,
 		AuthPassword: conf.AuthPassword,
 	}
-	server, err := charmstore.NewServer(db, cfg, charmstore.V4)
+	server, err := charmstore.NewServer(db, cfg, charmstore.Legacy, charmstore.V4)
```
Comment (urosj) [inline cmd/charmd/main.go:51]: Good catch!

File: internal/debug/handler.go [hunk 2]
```diff
@@ -1,3 +1,6 @@
+// Copyright 2014 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
```
Comment (urosj) [inline internal/debug/handler.go:3]: Thank you.

File: params/params.go [hunk 3]
```diff
@@ -12,6 +12,10 @@ import (
 	"gopkg.in/juju/charm.v3"
 )
 
+// BzrDigestKey is the extra-info key used to store the Bazaar digest
+// of an entity ingested from Launchpad.
+const BzrDigestKey = "bzr-digest"
+
```
Comment (urosj) [inline params/params.go]: Suggest to move this to lpad.go.

**Comment by urosj** [comment]
:+1: Lovely!

File: params/params.go (hunk 3 above)
Comment (frankban) [inline params/params.go]: Done.

**Comment by frankban** [comment]
Thanks for the reviews!
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/charmstore-merge

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results: http://ci.jujugui.org:8080/job/charmstore/269/

**Pull request merged**